src/index.js

```diff
@@ -1,10 +1,3 @@
-import {reducer} from './normalize/duck';
-import _Cashay from './Cashay';
-import _HTTPTransport from './HTTPTransport';
-
-export const Cashay = _Cashay;
-export const cashayReducer = reducer;
-export const HTTPTransport = _HTTPTransport;
-
-
-
+export {reducer} from './normalize/duck';
```

Comment:
change to `{reducer: cashayReducer}` and LGTM. Chances are folks have more than 1 reducer, so that'd force folks to alias their import statement, so i think it's nice for 3rd party packages to export something a little more descriptive.